src/main/resources/db/changelog/changesets/changelog_20241127T093705Z.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="lesaulnierkev (generated)" id="1732700251866-12">
+        <addColumn tableName="import_parameters">
+            <column name="root_network_entity_id" type="uuid"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="lesaulnierkev (generated)" id="1732700251866-13">
+        <sqlFile
+                dbms="postgresql"
+                encoding="UTF-8"
+                path="migrationImportParamsToRootNetworkPostgres.sql"
+                relativeToChangelogFile="true"
+                splitStatements="true"
+                stripComments="true"/>
+        <sqlFile
+                dbms="!postgresql"
+                encoding="UTF-8"
+                path="migrationImportParamsToRootNetworkH2.sql"
+                relativeToChangelogFile="true"
+                splitStatements="true"
+                stripComments="true"/>
+    </changeSet>
+    <changeSet author="lesaulnierkev (generated)" id="1732700251866-14">
+        <addNotNullConstraint tableName="import_parameters" columnName="root_network_entity_id"/>
+    </changeSet>
+    <changeSet author="lesaulnierkev (generated)" id="1732700251866-16">
+        <createIndex indexName="IX_import_parametersPK" tableName="import_parameters" unique="true">
```

Comment:
The index of the primary key is already created !
![Capture d’écran du 2024-11-27 15-37-41](https://github.com/user-attachments/assets/977dfc14-4586-40f0-824f-03c91e005fd7)

Reply:
fixed